modules/configuring/proc-mounting-additional-files-in-your-custom-configuration-using-rhdh-operator.adoc

```diff
@@ -74,9 +73,8 @@ spec:
           mountPath: /my/my-rhdh-secret/path
 
 ----
-====
-
++
 [NOTE]
-====
+----
```

Comment:
does DITA not support regular admonition formatting with `====`?

Reply:
I rolled it back. Now the Vale check is live we should see openly how it looks.

Reply:
```suggestion
====
```